html-forms/index.html

```diff
@@ -6,29 +6,44 @@
 <body>
   <h1>Coding Form!</h1>
   <form action="">
-    <div>Email
-      <input type="text" name="email"/>
+    <div class="email">
+      <label for="email">Email</label>
+    </div>
+    <input type="text" name="email" id="email"/>
+
+    <div class="why is coding cool">
+      <label for="why is coding cool">Why is coding cool?</label>
```

Comment:
Make sure the value of the `for` attribute matches the value of the `id` attribute on the form control this matches too. `for` and `id` values also shouldn't include spaces, you can separate words with dashes instead, `like-this`